pkg/instaclustr/client.go

```diff
@@ -2215,6 +2215,31 @@ func (c *Client) UpdateClusterSettings(clusterID string, settings *models.Cluste
 	return nil
 }
 
+func (c *Client) CheckIfAWSEndpointServicePrincipalExists(id string) (bool, error) {
+	url := c.serverHostname + AWSEndpointServicePrincipalEndpoint + "/" + id
```

Comment:
`AWSEndpointServicePrincipalEndpoint` already has a backslash at the end, why did you add an additional one?